execution/executor/src/components/apply_chunk_output.rs

```diff
@@ -330,6 +331,11 @@ impl ApplyChunkOutput {
             let (write_set, events, per_txn_reconfig_events, gas_used, status) =
```

Comment:
is the per_txn_reconfig_events still useful here?

Reply:
still used in below:
```
            let txn_to_commit = TransactionToCommit::new(
                txn,
                txn_info,
                state_updates,
                write_set,
                events,
                !per_txn_reconfig_events.is_empty(),
            );
```